com.github.TheThirdOne.rars.yml

```diff
@@ -0,0 +1,81 @@
+app-id: com.github.TheThirdOne.rars
+
+command: /app/bin/rars
+
+finish-args:
+  - --env=PATH=/app/jre/bin:/usr/bin
+  - --socket=x11
+  - --socket=fallback-x11
+  - --filesystem=xdg-desktop:rw
+  - --filesystem=xdg-documents:rw
+
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  # See https://github.com/flathub/org.freedesktop.Sdk.Extension.openjdk17#usage
+  - org.freedesktop.Sdk.Extension.openjdk17
+
+modules:
+  # See https://github.com/flathub/org.freedesktop.Sdk.Extension.openjdk17#usage
+  - name: openjdk
+    buildsystem: simple
+    build-commands:
+      - /usr/lib/sdk/openjdk17/install.sh
+
+  - name: rars
+    buildsystem: simple
+    sources:
+      # See https://github.com/TheThirdOne/rars/releases
+      - type: file
+        url: https://github.com/TheThirdOne/rars/releases/download/v1.5/rars1_5.jar
+        dest-filename: rars.jar
+        sha512: 0644b78691167ba985724e7f545d44f7a0a64f2dc2b9e1b9544a7a4f56e58cbc7ba089bef4e0a7281528de3a5aa943b3be792782fa262211ec596a8104f0fabf
+      - type: script
+        commands:
+          # Get the screen width (See https://superuser.com/a/1567163)
+          - SCREEN_WIDTH=$(cat /sys/class/graphics/*/virtual_size | grep -Eow -m 1 "^([0-9]+)")
```

Comment:
are you sure this work?

Reply:
Yes - I have tested it on two different systems and it works fine. This will be removed when [this issue](https://github.com/TheThirdOne/rars/issues/124) is closed. In the meantime, it makes the package much more plug-and-play than if you always had to manually specify the `RARS_SCALE` environment variable.